content/applications/productivity/documents.rst

```diff
@@ -4,392 +4,346 @@ Documents
-:guilabel:`Condition type`. Once done, click :guilabel:`New Rule`.
-
-To create a rule, you typically select a :guilabel:`field`, an :guilabel:`operator`, and a
-:guilabel:`value`. For example, if you want to add a workflow action to all the PDF files inside a
-workspace, set the :guilabel:`field` to *Mime Type*, the :guilabel:`operator` to *contains*, and the
-pdf :guilabel:`value`.
-
-.. image:: documents/domain-condition-example.png
-   :alt: Example of a workflow action's domain condition in Odoo Documents
-
-Click the :icon:`fa-plus` (:guilabel:`Add New Rule`) icon and the :icon:`fa-sitemap`
-(:guilabel:`Add branch`) icon to add conditions and sub-conditions. You can then specify if your
-rule should match :guilabel:`all` or :guilabel:`any` conditions. You can also edit the rule directly
-using the :guilabel:`Code editor`.
-
-Configure the actions
----------------------
-
-Go to the :guilabel:`Actions` section to set up your action. You can simultaneously:
-
-- **Move to Workspace**: move the file to any workspace;
-- **Create**: create one of the following items attached to the file in your database:
-
-  - **Link to record**: create a link between a document and a record from a specific model;
-  - **Product template**: create a product you can edit directly;
-  - **Task**: create a Project task you can edit directly;
-  - **Signature PDF template**: create a new Sign template to send out;
-  - **PDF to sign**: create a Sign template to sign directly;
-  - **Applicant**: create a new HR application you can edit directly;
-  - **Vendor bill**: create a vendor bill using OCR and AI to scrape information from the file
-    content;
-  - **Customer invoice**: create an invoice using OCR and AI to scrape information from the file;
-  - **Vendor credit note**: create a vendor credit note using OCR and AI to scrape information
-    from the file;
-  - **Credit note**: create a customer credit note using OCR and AI to scrape information from the
-    file;
-  - **Miscellaneous Operations**: create an entry in the Miscellaneous Operations journal;
-  - **Bank Statement**: import a bank statement;
-  - **Purchase Receipt**: create a vendor receipt;
-  - **Expense**: create an HR expense.
-
-- **Set Contact**: add a contact to the file, or replace an existing contact with a new one;
-- **Set Owner**: add an owner to the file, or replace an existing owner with a new one;
-- **Set Tags**: add, remove, and replace any number of tags.
-
-To set up activities related to your actions, go to the :guilabel:`Activities` section:
-
-- **Activities - Mark all as Done**: mark all activities linked to the file as done;
-- **Activities - Schedule Activity**: create a new activity linked to the file as configured in
-  the action. You can choose to set the activity on the document owner.
-
-Digitize documents with AI and optical character recognition (OCR)
+File digitization using AI and optical character recognition (OCR)
```

Comment:
To what extent do we use AI, maybe we can just leave 
`File digitization with optical character recognition.` or 'File digitization with AI` 

I feel like both is a lot since OCR is just the 'tool'